mods/d2k/chrome.yaml

```diff
@@ -100,6 +100,7 @@ flags: buttons.png
 	smuggler: 111,114,23,21
 	fremen: 133,114,23,21
 	worm: 156,114,23,21
+	mercenary: 179,114,20,21
```

Comment:
I thought it looks better having it as `20x21` (symmetrical):
![mercflags](https://cloud.githubusercontent.com/assets/7704140/22406810/f38d19ba-e659-11e6-8619-e77627374314.png)